### DISCUSSION TITLE
Inpainting is currently non-functional on the Mac OSX platform.Fooocus version: 2.1.824

### DISCUSSION THREAD

**Discussion by QuantumScripter** [lllyasviel/Fooocus#1342] [General]
It starts for 2 seconds, then NOTHING!

**Comment by lllyasviel** [comment]
please take a look at https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md before making a post

**Reply by QuantumScripter** [reply]
Thanks a lot for the reply ! I tried to run it with rundiffusion on macosx , still no luck ! please help !
